validator/keymanager/remote-web3signer/client.go

```diff
@@ -2,34 +2,41 @@ package remote_web3signer
 
 import (
 	"bytes"
-	"encoding/hex"
 	"encoding/json"
 	"fmt"
 	"io"
 	"net/http"
 	"net/url"
-	"strings"
 	"time"
 
+	"github.com/ethereum/go-ethereum/common/hexutil"
 	"github.com/pkg/errors"
 	"github.com/prysmaticlabs/prysm/crypto/bls"
+	"github.com/prysmaticlabs/prysm/encoding/bytesutil"
 )
 
 const (
 	ethApiNamespace = "/api/v1/eth2/sign/"
 	maxTimeout      = 3 * time.Second
 )
 
+// Web3SignerClient defines the interface for interacting with a remote web3signer.
+type Web3SignerClient interface {
+	Sign(pubKey string, request *SignRequest) (bls.Signature, error)
+	GetPublicKeys(url string) ([][48]byte, error)
+	//ReloadSignerKeys() error
+	//GetServerStatus() (string, error)
```

Comment:
Removed commented code. If needed, we can add it later

We should also rename this to be more clear about what it does
`httpSignerClient interface` perhaps. It should probably also be unexported